dapp/package.json

```diff
@@ -49,13 +50,18 @@
     "webpack-dev-server": "^3.1.3"
   },
   "dependencies": {
+    "@colony/colony-js-adapter-ethers": "^1.3.0",
+    "@colony/colony-js-client": "^1.3.0",
+    "@colony/colony-js-contract-loader-http": "^1.3.0",
     "classnames": "^2.2.5",
+    "ethers": "^3.0.21",
     "react": "^16.3.2",
     "react-dom": "^16.3.2",
     "react-redux": "^5.0.7",
     "react-router": "^4.2.0",
     "react-router-redux": "^5.0.0-alpha.9",
     "redux": "^4.0.0",
-    "redux-actions": "^2.3.0"
+    "redux-actions": "^2.3.0",
+    "redux-thunk": "^2.3.0"
```

Comment:
redux-thunk is included in both devDependencies & dependencies. Additionally they'er two different versions.

Can we just keep the "dependencies" entry and get rid of the other?

Reply:
Strange, I tried removing the entry in package-lock.json, but it re-adds it when I npm install. There must be some other package that's dependent on the older version.

Reply:
Which one did you remove? I think we should remove `@types/redux-thunk` as it's deprecated, see this: https://www.npmjs.com/package/@types/redux-thunk